mozilla/activity-stream
# Preparing the release of Activity Stream v1.6.0

Hi all,

We are now preparing the release of A-S 1.6.0. The add-on has been made available at [pre-release channel](https://moz-activity-streams-prerelease.s3.amazonaws.com/dist/latest.html). Please install it, test it, and leave the feedback, bug reports, and other inputs in this thread.

Highlights&Changelog:


# Release checklist
- QA
- Product
- Engineering

Please use the `release/block` or `release/uplift` to label the bug reports as well as other release related issues. Once you finish the review, please check off the corresponding item in the above checklist.

*Note*: checking off the item may not work, you can either modify it in place (i.e modify `[ ]` to `[x]`), or just drop a good old `r+` comment below.

*Note*: release highlights and the changelog will be attached shortly.

Thanks
